npm install doesn't work

When I run npm install, I get the following output:
```
MacBook-Pro-Micha:can-validate mickmcgrath$ npm install
npm WARN deprecated lodash@2.4.1: lodash@<3.0.0 is no longer maintained. Upgrade to lodash@^4.0.0.
npm WARN deprecated js-beautify@1.3.4: major indenting and html bug fixes in 1.4.0.
npm WARN deprecated MD5@1.3.0: deprecated, use lowercase 'md5@2.x' from now on
npm WARN deprecated lodash@1.0.2: lodash@<3.0.0 is no longer maintained. Upgrade to lodash@^4.0.0.
npm WARN deprecated graceful-fs@1.2.3: graceful-fs version 3 and before will fail on newer node releases. Please update to graceful-fs@^4.0.0 as soon as possible.
npm WARN deprecated graceful-fs@3.0.8: graceful-fs version 3 and before will fail on newer node releases. Please update to graceful-fs@^4.0.0 as soon as possible.
 
> fsevents@1.0.11 install /Users/mickmcgrath/Projects/Bitovi/can-validate/node_modules/documentjs/node_modules/chokidar/node_modules/fsevents
> node-pre-gyp install --fallback-to-build

[fsevents] Success: "/Users/mickmcgrath/Projects/Bitovi/can-validate/node_modules/documentjs/node_modules/chokidar/node_modules/fsevents/lib/binding/Release/node-v14-darwin-x64/fse.node" is installed via remote
npm WARN deprecated babel-core@5.8.14: Babel 5 is no longer being maintained. Upgrade to Babel 6.
npm WARN engine hawk@0.10.2: wanted: {"node":"0.8.x"} (current: {"node":"0.12.5","npm":"2.14.9"})
 
> utf-8-validate@1.2.1 install /Users/mickmcgrath/Projects/Bitovi/can-validate/node_modules/documentjs/node_modules/steal-tools/node_modules/ws/node_modules/utf-8-validate
> node-gyp rebuild

  CXX(target) Release/obj.target/validation/src/validation.o
  SOLINK_MODULE(target) Release/validation.node
npm WARN engine hoek@0.7.6: wanted: {"node":"0.8.x"} (current: {"node":"0.12.5","npm":"2.14.9"})
npm WARN engine boom@0.3.8: wanted: {"node":"0.8.x"} (current: {"node":"0.12.5","npm":"2.14.9"})
npm WARN engine sntp@0.1.4: wanted: {"node":"0.8.x"} (current: {"node":"0.12.5","npm":"2.14.9"})
npm WARN engine cryptiles@0.1.3: wanted: {"node":"0.8.x"} (current: {"node":"0.12.5","npm":"2.14.9"})
 
> bufferutil@1.2.1 install /Users/mickmcgrath/Projects/Bitovi/can-validate/node_modules/documentjs/node_modules/steal-tools/node_modules/ws/node_modules/bufferutil
> node-gyp rebuild

  CXX(target) Release/obj.target/bufferutil/src/bufferutil.o
  SOLINK_MODULE(target) Release/bufferutil.node
npm WARN deprecated line-numbers@0.2.0: Copy its ~20 LOC directly into your code instead.
 
> documentjs@0.3.6 postinstall /Users/mickmcgrath/Projects/Bitovi/can-validate/node_modules/documentjs
> install-engine-dependencies

npm ERR! git fetch -a origin (git://github.com/bitovi/source-map.git) fatal: unable to connect to github.com:
npm ERR! git fetch -a origin (git://github.com/bitovi/source-map.git) github.com[0: 192.30.252.128]: errno=Operation timed out
npm ERR! git fetch -a origin (git://github.com/bitovi/systemjs.git) fatal: unable to connect to github.com:
npm ERR! git fetch -a origin (git://github.com/bitovi/systemjs.git) github.com[0: 192.30.252.128]: errno=Operation timed out
npm ERR! Darwin 14.5.0
npm ERR! argv "node" "/usr/local/bin/npm" "install"
npm ERR! node v0.12.5
npm ERR! npm  v2.14.9
npm ERR! code 128

npm ERR! Command failed: git fetch -a origin
npm ERR! fatal: unable to connect to github.com:
npm ERR! github.com[0: 192.30.252.128]: errno=Operation timed out
npm ERR! 
npm ERR! 
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/mickmcgrath/Projects/Bitovi/can-validate/npm-debug.log
```


This is blocking https://github.com/canjs/can-validate/issues/20

--- a/can-validate.test.js
+++ b/can-validate.test.js
@@ -1,4 +1,9 @@
-import 'can-validate/validate.test';
+
+//TODO: We can't use two different shims at the same time. But, this first test
+// uses a custom shim since we are simply testing the core script and not the shims.
+// Including it with the rest of the tests causes the shim's to override each other
+// Need to find a workaround for this.
+//import 'can-validate/validate.test';
 import 'can-validate/map/validate/validate.test';
 import 'can-validate/shims/validatejs.test';
 import 'can-validate/shims/validations.test';

--- a/can-validate/map/validate/validate.test.js
+++ b/can-validate/map/validate/validate.test.js
@@ -1,6 +1,10 @@
 /* jshint asi: false */
 import can from 'can';
-import 'can-validate/map/validate/validate';
+import 'can/map/define/';
+import 'can-validate/can-validate';
+import 'can-validate/map/validate/';
+import 'can-validate/shims/validatejs.shim';
+
 import 'steal-qunit';
 
 var ValidatedMap = can.Map.extend({
@@ -14,6 +18,10 @@ var ValidatedMap = can.Map.extend({
 	}
 });
 
-var test = new ValidatedMap();
+var validatedMap = new ValidatedMap();
 
-test.attr('myNumber', null);
+test('validate on set', function () {
+	validatedMap.attr('myNumber', '');
+
+	equal(validatedMap.errors.myNumber.length, 1, 'Validate ran successfully on set.');
+});

--- a/can-validate/shims/validatejs.shim.js
+++ b/can-validate/shims/validatejs.shim.js
@@ -54,7 +54,7 @@ var Shim = can.Construct.extend({
 	once: function (value, options, name) {
 		var errors = [];
 		var opts = [];
-		var validationOpts = []
+		var validationOpts = [];
 
 		// Check if name was passed, determines which validate method to use
 		if (name) {

--- a/can-validate/shims/validatejs.test.js
+++ b/can-validate/shims/validatejs.test.js
@@ -1,6 +1,9 @@
 /* jshint asi: false */
 import can from 'can';
-import 'can-validate/map/validate/validate';
+import 'can/map/define/';
+import 'can-validate/can-validate';
+import 'can-validate/map/validate/';
+import 'can-validate/shims/validatejs.shim';
 import 'steal-qunit';
 
 var ValidatedMap = can.Map.extend({
@@ -23,14 +26,14 @@ var ValidatedMap = can.Map.extend({
 	}
 });
 
-var test = new ValidatedMap({});
+var validatedMap = new ValidatedMap({});
 
 test('validate on init', function () {
-	equal(test.errors.myNumber.length, 1, 'Validate ran successfully on init.');
+	equal(validatedMap.errors.myNumber.length, 1, 'Validate ran successfully on init.');
 });
 
 test('validate', function () {
-	equal(test.errors.myString, undefined, 'Does not validate on init by default.');
-	test.attr('myString', 'a');
-	equal(test.errors.myString.length, 1, 'Validate ran successfully on change of value.');
+	equal(validatedMap.errors.myString, undefined, 'Does not validate on init by default.');
+	validatedMap.attr('myString', '');
+	equal(validatedMap.errors.myString.length, 1, 'Validate ran successfully on change of value.');
 });

--- a/can-validate/shims/validations.test.js
+++ b/can-validate/shims/validations.test.js
@@ -14,6 +14,6 @@ var ValidatedMap = can.Map.extend({
 	}
 });
 
-var test = new ValidatedMap();
+var validatedMap = new ValidatedMap();
 
-test.attr('myNumber', null);
+validatedMap.attr('myNumber', null);

--- a/package.json
+++ b/package.json
@@ -32,15 +32,15 @@
   },
   "homepage": "https://github.com/canjs/can-validate",
   "devDependencies": {
-    "can": "^2.3.0-pre.2",
-    "documentjs": "^0.3.0-pre.5",
-    "jquery": "^2.1.4",
-    "steal": "^0.10.5",
-    "steal-qunit": "0.0.4",
-    "steal-tools": "^0.11.0-pre.1",
-    "testee": "^0.2.0",
-    "validate.js": "^0.8.0",
-    "xo": "^0.9.0"
+    "can": "^2.3.23",
+    "documentjs": "^0.4.4",
+    "jquery": "^2.2.3",
+    "steal": "^0.16.2",
+    "steal-qunit": "0.1.1",
+    "steal-tools": "^0.16.1",
+    "testee": "^0.2.5",
+    "validate.js": "^0.9.0",
+    "xo": "^0.14.0"
   },
   "browser": {
     "./can-validate": "./dist/cjs/can-validate/can-validate",

--- a/package.json
+++ b/package.json
@@ -35,6 +35,7 @@
     "can": "^2.3.23",
     "documentjs": "^0.4.4",
     "jquery": "^2.2.3",
+    "qunitjs": "^1.23.1",
     "steal": "^0.16.2",
     "steal-qunit": "0.1.1",
     "steal-tools": "^0.16.1",

